tests/dummy/app/routes/nodes/detail/draft-registrations.js

```diff
@@ -5,29 +5,28 @@ export default Ember.Route.extend({
     model() {
         let node = this.modelFor('nodes.detail');
         let drafts = node.get('draftRegistrations');
-        return Ember.RSVP.hash({
-            node: node,
-            drafts: drafts
-        });
+        return drafts;
+    },
+    setupController(controller, model) {
+        this._super(controller, model);
+        controller.set('node', this.modelFor('nodes.detail'));
     },
     actions: {
         createDraft(schemaId) {
-            var node = this.modelFor(this.routeName).node;
+            var _this = this;
```

Comment:
I don't see why `_this` is needed here-- shouldn't regular `this` work the same?